docs/development.md

```diff
@@ -1,128 +1,119 @@
-`styleguide.config.js` file. The most common use case for this is creating a
-sidebar layout custom to the packages directory structure.
+- `├── examples/` – visual component examples and generated documentation
+- `├── src/` – component source code and spec tests<br>
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`├── elements/` – high abstraction components that wrap interaction behavior and visual styling<br>
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`├── styled/` – view-level [styled-components](https://styled-components.com/) that contain theme based CSS<br>
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`└── index.ts` – public-facing component exports for this package
+- `├── package.json` – configuration for the package published to the registry
+- `├── README.md` – essential package description, install, and usage instructions
 
-To include examples with your code include a markdown file. `FooComponent.js`
-would be documented with `FooComponent.example.md`.
+<!-- markdownlint-enable -->
 
-To start the documentation in development mode use the `yarn start` command.
+The entire Garden React codebase is statically type-checked using
+[TypeScript](https://www.typescriptlang.org/). See the [API](api.md)
+documentation for in-depth treatment of Garden's Container-View-Element
+architecture, along with rules that apply to each component type.
 
-```sh
-# Prompts for an individual package
-yarn start
-
-# Starts the individual package immediately
-yarn start --scope @zendeskgarden/react-buttons
-```
+## Package examples
 
-## Component Requirements
+Package example documentation is generated via [React
+Styleguidist](https://react-styleguidist.js.org/). Run `yarn start` to build
+and serve package documentation in development mode (with hot reloading). The
+[global configuration](/utils/styleguide/styleguide.base.config.js) is
+extended by a package-local `styleguide.config.js` which determines section
+structure and content for each package page.
 
-### Elements
+In general, an example page includes:
 
-All elements must
+- the package README
+- a "knobs" style example that toggles various visual component states
+- API prop sheets for all public element exports
 
-- Be implemented with associated `Container` and `View` components
-- Provide `uncontrolled` and `controlled` state management if necessary
-  - Be implemented with the `ControlledComponent` state abstractions if necessary
-- Create an abstraction to benefit a majority of use cases
+Include additional example files to demonstrate complex component behaviors
+or common interactions with other Garden components.
 
-### Containers
+## Package testing
 
-All containers must
+The Garden `react-components` repo is strongly tested – holding steady at
+~95% [coverage](https://coveralls.io/github/zendeskgarden/react-components).
+DOM testing is implemented using the popular and pragmatic [React Testing
+Library](https://testing-library.com/react). [Jest](https://jestjs.io/) is
```

Comment:
There is a lot of excess descriptives here and we shouldn't mention a fix coverage percentage. This will change overtime. Maybe something like:

"This repo is tested using [React Testing Library](#)."

I'm not even sure we need to mention Jest.